ten_quick_tips_live_coding.md

```diff
@@ -152,58 +154,39 @@ and these may help learners to understand the stages of the lesson and to organi
 What can work really well is when you as instructor generate the illustrations on the whiteboard as you progress through the material.
 This allows you to build up diagrams,
 making them increasingly complex in parallel with the material you are teaching.
-[FIXME: mention dual coding and the value of presenting complementary information on visual and linguistic channels.]
+Presenting complementary information using visual and verbal representations helps learning (so-called "dual coding") (Clark, J.M. & Paivio, A. Educ Psychol Rev (1991) 3: 149. https://doi.org/10.1007/BF01320076).
 
 Diagramming helps learners understand the material,
-makes for a more lively workshop (you'll have to move between your computer and the blackboard),
+makes for a more lively workshop (you'll have to move between your computer and the whiteboard),
 and gathers the learners' attention to you as well.
-[FIXME: more detail here?]
 
 ## Tip 7. Stick to the lesson material.
 
-Participatory live coding when teaching works best with a well-developed and tested script.
-[FIXME: GVW doesn't like the term "script". Worth making the analogy with jazz - there's a score, but you're expected to improvise on top of it?]
+Participatory live coding when teaching works best with a well-developed and tested lesson.
 It may be tempting to deviate from your material because you would like to show a neat trick
 or demonstrate some alternative way of doing something,
-but there is always a fair chance you'll run into something unexpected that you then have to explain,
-so it is advised not to improvise.
-[FIXME: GVW disagrees :-)]
-
-If you really want to use something outside of the material,
-try it out thoroughly before teaching it:
-run through the lesson as you would during the actual teaching and test the effect of your modification.
-
+but there is always a fair chance you'll run into something unexpected that you then have to explain. It is thus advised not to improvise until you are familiar enough with the material and want to explain something with a low risk of failure.
 Use printouts of the lesson material during teaching,
 or alternatively use a second device (tablet or laptop) on which you can view your notes.
-[FIXME: mention timers here - when you tell learners they have 5 minutes for an exercise, keep yourself honest.]
-
-[FIXME: create a backlog of "what ifs?" and other questions that you can sort through and think about while learners are doing exercises.]
+Consider the use of a timer for exercises: they help keep yourself honest when you tell learners they have 5 minutes for an exercise.
+Sometimes a question or a "what ifs?" comes up that you'd like to address but need some time to sort through. Collect these, for example on sticky notes, think about while learners are doing exercises and answer them afterwards. You then don't disrupt the flow of the lesson while still showing that you take the learner's questions seriously.
```

Comment:
Do we mention use of a shared Google Doc anywhere? I now ask learners to add backlog questions to the bottom of the doc while I'm teaching or they're doing exercises rather than trying to do sticky notes.

Reply:
+1, but again I don't want to add specific tools